package.json

```diff
@@ -9,7 +9,11 @@
     "lint": "lerna run lint",
     "start": "lerna run start",
     "bootstrap": "lerna bootstrap",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "ngrok": "ngrok start --config=./ngrok/ngrok.yml --all",
+    "ngrok:setup": "python3 ./ngrok/ngrok-setup.py",
+    "ngrok:api": "ngrok start --config=./ngrok/ngrok.yml api",
+    "ngrok:webapp": "ngrok start --config=./ngrok/ngrok.yml webapp"
```

Comment:
These new commands should be documented in the README, as well as the need to have python 3 installed on your machine to run them